SITCOM-1397_Extract_dominant_frequencies_from_sound_waves_in_video_associated_with_vibrations_on_Top-End_Assembly_II.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
```

Comment:
<p>I suggest to extract the mono signal from the beginning and to limit the analysis to this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
The beginning part is deleted